app/views/revenue_estimations/_form.html.erb

```diff
@@ -30,12 +30,12 @@
   <div class="form-row">
     <div class="form-group col-6">
       <%= f.label :month, :class => 'control-label' %>
-      <%= f.number_field :month, :class => 'form-control', value: f.object.date&.month %>
+      <%= f.select :month, options_for_select((1..12).map { |m| [I18n.t("date.month_names")[m], m] }, f.object.date&.month), {}, class: 'form-control' %>
```

Comment:
I liked it 👏🏻 , in another issue I will default it to the current month. Not necessary for now.

Reply:
Just did it! Haha